pkg/utils/gpu.go

```diff
@@ -33,7 +33,7 @@ const (
 )
 
 func GetAKSGPUImageSHA(size string) string {
-	if useGridDrivers(size) {
+	if UseGridDrivers(size) {
 		return AKSGPUGridSHA
 	}
```

Comment:
Do these values have to come from us? if we use the ones from agentbaker and default them in NPS we can have consistency and won't have to bump the shas for cves

Reply:
We are not using SHAs. Just that `UseGridDrivers()`. But could still be a candidate for that consideration.